static/js/user_groups.ts

```diff
@@ -176,3 +176,26 @@ export function get_recursive_subgroups(user_group_id: number): Set<number> | un
     }
     return subgroup_ids;
 }
+
+export function is_user_in_group(user_group_id: number, user_id: number): boolean {
+    const user_group = user_group_by_id_dict.get(user_group_id);
+    if (user_group === undefined) {
+        blueslip.error(`Could not find user group with ID ${user_group_id}`);
+        return false;
+    }
+    if (is_direct_member_of(user_group_id, user_id)) {
```

Comment:
Looking at this, it really feels like the parameters for `is_direct_member_of` are backwards -- `user_id` is a member of `user_group_id`, not the other way around. Do you think it's worth doing another prep commit to swap the parameter order for this function?